src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -72,9 +72,9 @@ public Person getPersonById(int id)  {
 			p = dao.getOne(id);
 			System.out.println("This is my P: "+p);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)